save_text_report.py

```
@@ -0,0 +1,136 @@
+    report.append("ANALYSIS COMPLETE")
+    report.append("=" * 80)
+    
+    # Save to file
+    with open(output_file, 'w', encoding='utf-8') as f:
+        f.write('\n'.join(report))
+    
+    print(f"Report saved: {output_file}")
+    
+    # Open file
+    import os
+    os.startfile(str(output_file.absolute()))
```

Ruff (0.14.5)

Starting a process without a shell

(S606)